src/Settings/Identity/Index.jsx

```diff
@@ -1,8 +1,12 @@
-const [showBanner, setShowBanner] = useState(false);
+const [showBanner, setShowBanner] = useState(true);
 
 // const modalOpen = fractalModal?.alert ?? false;
 // const fractalModal = Storage.get("fractal-alert") ?? null;
```

Comment:
Does it make sense to get rid of these commented out lines?

Reply:
Yes. I fixed that logic related to modal